scene/gui/text_edit.cpp

```diff
@@ -4979,11 +4971,8 @@ String TextEdit::get_word(int p_line, int p_column) const {
 }
 
 Point2i TextEdit::get_line_column_at_pos(const Point2i &p_pos, bool p_clamp_line, bool p_clamp_column) const {
-	float rows = p_pos.y - theme_cache.style_normal->get_margin(SIDE_TOP);
-	if (!editable) {
-		rows -= theme_cache.style_readonly->get_offset().y / 2;
-		rows += theme_cache.style_normal->get_offset().y / 2;
-	}
+	Ref<StyleBox> style = _get_current_stylebox();
+	float rows = p_pos.y - (style->get_margin(SIDE_TOP) + (theme_cache.line_spacing / 2));
```

Comment:
Fixed a bug here where the half line spacing used to offset the text wasn't taken into account, check the video where the issue happens on the left side.

https://github.com/user-attachments/assets/070b313f-93f2-4da5-a4f2-daab37cea1ed

Note: This will change the return values for `get_pos_at_line_column` and `get_rect_at_line_column`.

Reply:
- This sounds kind of like https://github.com/godotengine/godot/issues/79735 but offset the other way.